api-reference/beta/api/filesecurityprofile_update.md

```diff
@@ -0,0 +1,127 @@
+# Update fileSecurityProfile
+
+Update the user-editable properties of a fileSecurityProfile object. **This method must include the fileSecurityProfileId as a parameter.**
+

```

Comment:
Same global comment here about the bold text regarding adding the parameter. If this needs to be stated, revise as per earlier feedback and remove the bold formatting.